plugins/microsoft_teams/icon_microsoft_teams/util/teams_utils.py

```diff
@@ -227,6 +227,53 @@ def send_html_message(
     return message
 
 
+def create_chat(connection: insightconnect_plugin_runtime.connection, members: str, topic: str) -> bool:
+    """
+    Creates a chat for given users
+
+    :param connection: Object (insightconnect_plugin_runtime.connection)
+    :param members: String
+    :return: boolean
```

Comment:
wrong return type here and in `list_messages_from_chat`?

Reply:
this has been updated to now correctly  use dict and list